packages/api/src/shared/config.ts

```diff
@@ -268,9 +268,7 @@ export class Config {
   static getSearchUsername(): string {
     return getEnvVarOrFail("SEARCH_USERNAME");
   }
-  static getSearchPassword(): string {
```

Comment:
passwords are secrets not env vars. see getSecret()

Reply:
The API does get secrets as env vars, see [api-service.ts](https://github.com/metriport/metriport/blob/5e3dea257861d8545b5edc8f8cc990a1c3426fbf/packages/infra/lib/api-stack/api-service.ts#L124).

If we're not using getting it from vars any longer, we could stop sending it over.